Added Django-Environ configs for dev

jbiason opened the pull request:
Also updated documentation to point how to use it.

luzfcb commented on README.md:
```diff
@@ -27,3 +27,42 @@ Mas mesmo entre os dev temos conhecimentos distintos, muitos da comunidade não
+
+Provavelmente irá aparecer em seu terminal algo como *(project-name)$*, agora vamos clonar o repositório do projeto:
+
+> git clone git@github.com:pythonbrasil/apyb.git
```
use a sintaxe https em vez de git, para evitar forçar ter que criar uma chave ssh e cadastra-la no github

luzfcb commented on README.md:
```diff
@@ -27,3 +27,42 @@ Mas mesmo entre os dev temos conhecimentos distintos, muitos da comunidade não
+
+Legal, agora já instalei todos os programas, vamos fazê-lo rodar em nosso computador?
+
+> make html  
```
eu inclui o script do pelican, `develop_server.sh`

basicamente é
```bash
./develop_server.sh start
./develop_server.sh stop
./develop_server.sh restart
```

o bom do develop_server.sh é que ele regera o site automaticamente, a cada modificalçao que é feita

humrochagf commented:
tá lindo de acompanhar essa parada hehe

jbiason committed:
Added Django-Environ configs for dev

Also updated documentation to point how to use it.

jbiason committed:
Reverse the dev config logic

Instead of having production configuration on the settings and having an .env
for development, use the default configuration as dev and allow production to
add its own .env.
